packages/web-component/test/helpers/templates.test.ts

```diff
@@ -12,26 +12,22 @@ describe('templates', () => {
   it('should handle descope inputs', async () => {
     document.body.innerHTML = `<div>
 			<input class="descope-input" name="email">
-			<textarea class="descope-input" name="description">
```

Comment:
Y?
Also down' we want to test the new components?

Reply:
It's testing the same thing so I removed it
Basically the code does not refer to specific components so it should be fine